GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -603,5 +615,78 @@ private void saveAsToolStripMenuItem1_Click(object sender, EventArgs e)
                 }
             }
         }
+
+
+        private bool DeleteSelectedDiffFiles()
+        {
+            if (DiffFiles.Focused)
+            {
+                return DeleteSelectedFiles();
+            }
+            return false;
+        }
+
+        private bool DeleteSelectedFiles()
```

Comment:
Please move this to `RevisionDiffController`
`Module` and any other dependencies should be injected into `RevisionDiffController`'s constructor (similar to `RevisionFileTreeController`)